store/index.js

```diff
@@ -1,114 +1,116 @@
-			jsonFile: null
-		},
-
-		// categorization.vue
-		categorization: {
-
-			// Category paint brushes
-			current: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"
-			},
-			default: {
-	
-				bColor: "white",
-				category: "",
-				fColor: "black"			
-			},
-
-			// Stores currently painted items
-			paintingData: {},
-
-			// Stores currently loaded table data
-			tableData: {}
-		},
-
-		// annotation.vue
-		annotation: {
-
-		},
-
-		// download.vue
-		download: {
-
-		}
-	},
-
-	painting: {
-		
-		classes: {
-
-			paint0: "category-style-0",
-			paint1: "category-style-1",
-			paint2: "category-style-2",
-			paint3: "category-style-3",
-			paint4: "category-style-4"
-		},
-
-		palette: {
-			backgroundColors: [],
-			foregroundColors: []
-		}
-	}
+        home: {
+            
+            accessibility: true,
+            fullName: "Home",
+            location: "/",
+            pageName: "index",
+        },
+
+        categorization: {
+            
+            accessibility: false,
+            fullName: "Categorization",
+            location: "categorization",
+            pageName: "categorization"
+        },
+
+        annotation: {
+            
+            accessibility: false,
+            fullName: "Annotation",
+            location: "annotation",
+            pageName: "annotation"
+        },
+
+        download: {
+            
+            accessibility: false,
+            fullName: "Download",
+            location: "download",
+            pageName: "download"
+        }
+    },
+
+    // TSV file
+
+    // Participants.tsv file data
+    // For format see 'convertTsvLinesToDict' in index.js
+    tsvDataOriginal: null,
+
+    // Stores table data in format ready for Bootstrap table
+    // This is an array of objects. See 'tableDataFromTsvAndOrJson' in
+    // categorization.vue for exact format
+    tsvDataTableFormatted: [],
+
+    // Data dictionary
+
+    // Original data dictionary file data
+    // (formerly home.jsonFile)
+    dataDictionaryOriginal: null,
+
+    // User-amended data dictionary file data
+    dataDictionaryAmended: {},	
+
+    // Hardcoded list of categories used on the categorization page
+    // and possibly elsewhere in the tool
+    // formerly (categorization page's recommendedCategories)
+    categoryList: [
+
+        "Subject ID",
+        "Age",
+        "Sex",
+        "Diagnosis",
+        "Assessment Tool"
+    ],	
+
+    // Mapping showing user-applied mappings between our categories
+    // and the columns in their tsv/data dictionary file(s)
+    // Keyed on column. If a column is not a key, it has not yet been linked to
+    // a category by the user
+    // (formerly categorization.paintingData)
+    categoryColumnMap: {},
+
+    // Maps our categories in 'categoryList' to colors in 'toolColorPalette'
+    // (Final class names pending). This way colors can be swappable and
+    // rearrangeable for categories
+    categoryToColorMap: {
```

Comment:
I believe we wanted to turn this into a computed property (via a hook on the upload page) and have an explicit mapping of category names to color names. This will allow us to access this more intuitively inside the app (e.g. by searching for the category name as the key rather than an index we have to get from some other place first.

Reply:
So in other words, depending on the flavor of annotation the category to color mapping changes? I think we discussed this but was not considered for this PR. Since I also recall us discussing putting that on a page that will precede the upload page, I think we should make an issue for this feature. I've created one here: https://github.com/metaneuro/annotation_tool/issues/67

Reply:
OK, makes sense to handle this in a separate PR. I mainly brought it up here because I find it more intuitive if `categoryToColorMap` has the actual category names (like `categoryColumnMap` does), i.e.:
```
categoryToColorMap: {
  "Age": "color1",
  "Sex": ...
}
```
I remember we discussed the computed property both in the context of future flexibility (for other categories), but also as a way to get the above explicit mapping without having to retype the category names a second time (and having to keep them in sync). I was more referring to the clarity aspect here. I'll add it to the issue.